css/style.css

```diff
@@ -0,0 +1,45 @@
+body {
+  background-image: linear-gradient(115deg, rgba(58, 58, 158, 0.8), rgba(136, 136, 206, 0.7)), url('../images/survey-form-background.jpeg');
+  font-family: 'Poppins', sans-serif;
+  font-size: 1rem;
+  font-weight: 400;
+  line-height: 1.4;
+  color: #fff;
+}
+
+.container {
+  max-width: 750px;
+  width: 100%;
+  margin: 3.125rem auto 0 auto;
+  box-sizing: border-box;
+}
+
+#survey-form {
+  background-image: linear-gradient(62deg, #3a3d40 0%, #181719 100%);
+  padding: 2.5rem 0.625rem;
+  border-radius: 0.25rem;
+}
+
+.form-input {
+  display: block;
+  width: 95%;
+  height: 2.375rem;
+  padding: 0.375rem 0.75rem;
+  color: #495057;
+  background-color: #fff;
+  background-clip: padding-box;
+  border: 1px solid #ced4da;
+  border-radius: 0.25rem;
+  transition: border-color 0.15s ease-in-out, box-shadow 0.15s ease-in-out;
+}
+
+.submit-button {
+  background-color: #50f703;
+  color: rgb(15, 0, 102);
+  padding: 2%;
+  font-size: 1.15rem;
+}
+
+#comments {
+  height: 120px;
+}
```

Comment:
Since your HTML code will be different for all the pages of your site but your CSS won't, it would be silly to reload them for every page. Separating CSS styles in separate files makes the cachable.